demo/pages/validation/index.html

```diff
@@ -34,5 +36,13 @@
 		             key="contact.address.state"
 		             length-equal="2">
 		</q-grid:rule>
+		<q-grid:rule for="column"
```

Comment:
do we need to changed for="column" to for="cell"?

Reply:
I can change, OK